app/views/citizens/edit.html.erb

```diff
@@ -1,4 +1,2 @@
-<h1>Editando Cidadão</h1>
-
-<%= render 'form', citizen: @citizen %>
+<%= render partial: 'citizens/form', locals: {action_message: t('links.edit') } %>
```

Comment:
Boas práticas Hash: adicionar espaço sempre após `{` e antes de `}`

Reply:
1fa5fdd74989c8a95225a0e94199504d62c836b0 and cebcf2845c5295d40bfd37969da0b8e10a058035